catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/OrderRequestImplTest.java

```diff
@@ -469,7 +501,7 @@ protected void writeFile(FileLocation destination, InputStream fileData, long si
         assertThat(holder.value.elements[0].files.length, is(1));
     }
 
-    @Test(expected = NO_IMPLEMENT.class)
+    @Test/*(expected = BAD_OPERATION.class)*/
```

Comment:
I was under the impression that getBadDestination was more focused on a bad email address, at which point the test would have to time out. Would an additional test be needed to throw the exception you mentioned with a call to h_dest?